metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-env.xml

```diff
@@ -177,7 +177,7 @@ indexing.executors=0
 ##### Kafka #####
 kafka.zk={{ zookeeper_quorum }}
 kafka.broker={{ kafka_brokers }}
-kafka.start=WHERE_I_LEFT_OFF
+kafka.start=UNCOMMITTED_EARLIEST
 ##### Indexing #####
```

Comment:
Little late to the party, but I think this is causing an error in Quick Dev. The basis assumption is that the MPack is unchanged for a given quick dev image, so kafka.start remains WHERE_I_LEFT_OFF right now but that's not part of the enum, so it throws. I think the easy thing is to create a new image. What do you think?